apps/console/src/features/users/components/user-roles-edit.tsx

```diff
@@ -801,10 +825,10 @@ export const UserRolesList: FunctionComponent<UserRolesPropsInterface> = (
                                                                             content="View permissions"
                                                                             position="top right"
                                                                             trigger={
-                                                                                <Icon
+                                                                                (<Icon
                                                                                     data-testid={
                                                                                         `user-mgt-roles-list-
-                                                                                        ${ userRole[ 1 ] }-
+                                                                                        ${userRole[1]}-
```

Comment:
Revert back

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3153/commits/313c92a74b1c03a09f6d70e4b3f1f44530c6c246